doc/source/whatsnew/v0.19.0.txt

```diff
@@ -417,6 +417,107 @@ New Behavior:
 
    type(s.tolist()[0])
 
+.. _whatsnew_0190.api.series_ops:
+
+``Series`` operators for different indexes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Following ``Series`` operators has been changed to make all operators consistent,
+including ``DataFrame`` (:issue:`1134`, :issue:`4581`, :issue:`13538`)
+
+- ``Series`` comparison operators now raise ``ValueError`` when ``index`` are different.
+- ``Series`` logical operators align both ``index``.
+
+As a result, ``Series`` and ``DataFrame`` operators behave consistently as below:
```

Comment:
add a comment on how this is more strict than previously. (maybe in a warning). IOW in cases where it previously worked, it will now error if the alignment is incorrect. This will now raise rather than silently pass.